src/app/Scenes/HomeView/Sections/SalesRailHomeViewSection.tsx

```diff
@@ -0,0 +1,80 @@
+import { Flex, useScreenDimensions } from "@artsy/palette-mobile"
+import { SalesRailHomeViewSection_section$key } from "__generated__/SalesRailHomeViewSection_section.graphql"
+import { BrowseMoreRailCard } from "app/Components/BrowseMoreRailCard"
+import { CardRailFlatList } from "app/Components/Home/CardRailFlatList"
+import { SectionTitle } from "app/Components/SectionTitle"
+import { SalesRailItem } from "app/Scenes/HomeView/Sections/SalesRailItem"
+import { navigate } from "app/system/navigation/navigate"
+import { extractNodes } from "app/utils/extractNodes"
+import { useRef } from "react"
+import { FlatList } from "react-native-gesture-handler"
+import { graphql, useFragment } from "react-relay"
+
+interface SalesRailHomeViewSectionProps {
+  section: SalesRailHomeViewSection_section$key
+}
+
+export const SalesRailHomeViewSection: React.FC<SalesRailHomeViewSectionProps> = ({ section }) => {
+  const listRef = useRef<FlatList<any>>()
+  const data = useFragment(fragment, section)
+  const component = data.component
+  const sales = extractNodes(data.salesConnection)
+
+  const { width } = useScreenDimensions()
+  const isTablet = width > 700
+
```

Comment:
Although we always have sales, it will be nice to have a safeguard in case we don't have any
```suggestion
if (sales.length === 0) {
  return null
}
```